README.md

```diff
@@ -7,6 +7,10 @@ This is my Microverse personal portfolio project. Through this project I will de
 - Major languages: HTML, CSS
 - Technologies used: GitHub Actions
 
+## Deployment
+
+To run this project in your local environment, you can use this link; https://dodoburner.github.io/Portfolio/
```

Comment:
- [ ] [OPTIONAL] Good job it will be nice if you make the URL a link (e.g: To run this project in your local environment, you can use this [link](https://dodoburner.github.io/Portfolio/)